src/main/java/net/atos/zac/util/OpenZaakReadinessHealthCheck.java

```diff
@@ -18,19 +16,22 @@
 
 import net.atos.client.zgw.ztc.ZtcClientService;
 import net.atos.client.zgw.ztc.model.CatalogusListParameters;
+import net.atos.zac.configuratie.ConfiguratieService;
 
 @Readiness
 @ApplicationScoped
 public class OpenZaakReadinessHealthCheck implements HealthCheck {
 
     private static final CatalogusListParameters CATALOGUS_LIST_PARAMETERS = new CatalogusListParameters();
 
-    static {
-        CATALOGUS_LIST_PARAMETERS.setDomein(CATALOGUS_DOMEIN);
-    }
+    private final ZtcClientService ztcClientService;
 
     @Inject
-    private ZtcClientService ztcClientService;
+    OpenZaakReadinessHealthCheck(ZtcClientService ztcClientService, ConfiguratieService configuratieService) {
+        this.ztcClientService = ztcClientService;
+
+        CATALOGUS_LIST_PARAMETERS.setDomein(configuratieService.readCatalogusDomein());
```

Comment:
Why this change? Static actually seems better to me or not? If we want to change this class I would say just convert the thing to Kotlin first. :-)

Reply:
Previously we had constant with the domain and now we have a function returning value.

Having the configuration service injected does not really allow for static initializer anymore, so I moved the init code to the constructor

Reply:
ah clear!